besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/BackupState.java

```diff
@@ -0,0 +1,124 @@
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.ParentCommand;
+
+@Command(
+    name = "x-backup-state",
+    description = "Backups up the state and accounts of a specified block.",
+    mixinStandardHelpOptions = true)
+public class BackupState implements Runnable {
+
+  @Option(
+      names = "--block",
+      paramLabel = MANDATORY_LONG_FORMAT_HELP,
+      description = "The block to perform the backup at (default: calculated chain head)",
+      arity = "1..1")
+  private final Long block = Long.MAX_VALUE;
+
+  @Option(
+      names = "--backup-path",
+      required = true,
+      paramLabel = MANDATORY_LONG_FORMAT_HELP,
+      description = "The path to store the backup files.",
+      arity = "1..1")
+  private final File backupDir = null;
+
+  @Option(
+      names = {"--compression-enabled"},
+      description = "Enable data compression",
+      arity = "1")
+  private final Boolean compress = true;
+
+  @ParentCommand private OperatorSubCommand parentCommand;
+
+  @Override
+  public void run() {
+    checkArgument(
+        parentCommand.parentCommand.dataDir().toFile().exists(),
+        "DataDir (the blockchain being backed up) does not exist.");
+    checkArgument(
+        backupDir.exists() || backupDir.mkdirs(),
+        "Backup directory does not exist and cannot be created.");
+
+    final BesuController besuController = createBesuController();
+    final MutableBlockchain blockchain = besuController.getProtocolContext().getBlockchain();
+    final WorldStateStorage worldStateStorage =
+        besuController.getProtocolContext().getWorldStateArchive().getWorldStateStorage();
+    final EthScheduler scheduler = new EthScheduler(1, 1, 1, 1, new NoOpMetricsSystem());
+    try {
+      final long targetBlock = Math.min(blockchain.getChainHeadBlockNumber(), this.block);
+      final StateBackupService backup =
+          new StateBackupService(
+              BesuInfo.version(), blockchain, backupDir.toPath(), scheduler, worldStateStorage);
+      final BackupStatus status = backup.requestBackup(targetBlock, compress, Optional.empty());
+
+      final double refValue = Math.pow(2, 256) / 100.0d;
+      while (status.isBackingUp()) {
+        if (status.getTargetBlockNum() != status.getStoredBlockNum()) {
+          System.out.printf(
+              "Chain Progress - %,d of %,d (%5.2f%%)%n",
+              status.getStoredBlockNum(),
+              status.getTargetBlockNum(),
+              status.getStoredBlockNum() * 100.0d / status.getTargetBlockNum());
+        } else {
+          System.out.printf(
+              "State Progress - %6.3f%% / %,d Accounts / %,d Storage Nodes%n",
+              status.getCurrentAccountBytes().toUnsignedBigInteger().doubleValue() / refValue,
+              status.getAccountCount(),
+              status.getStorageCount());
+        }
+        LockSupport.parkNanos(TimeUnit.SECONDS.toNanos(10));
+      }
+
+      System.out.printf(
```

Comment:
why we don't use LogManager here ?

Reply:
As a command the output should only ever be to standard out.  Logging does not guarantee this.